.github/release.yml

```diff
@@ -1,18 +1,20 @@
 changelog:
-   categories:
-    - title: Features
-       labels:
-         - RN: MAJOR FEATURE
-         - RN: MINOR FEATURE
-    - title: Fixes
-       labels:
-         - RN: BUGFIX
-         - RN: SAFETY IMPROVEMENT
-         - RN: IMPROVEMENT
-    - title: Targets
-       labels:
-         - RN: NEW TARGET
-         - RN: TARGET UPDATE
-    - title: Known Issues
-       labels: 
-         - BUG
+categories:
+- title: Features
+labels:
+- RN: MAJOR FEATURE
+- RN: MINOR FEATURE
+- title: Improvements
+labels:
+- RN: IMPROVEMENT
+- RN: SAFETY IMPROVEMENT
+- title: Fixes
+labels: 
+- RN: BUGFIX
+- title: Targets
+labels:
+- RN: NEW TARGET
+- RN: TARGET UPDATE
+- title: Known Issues
+labels:
+- BUG
```

Comment:
```suggestion
- BUG

```

Reply:
Added line break char (end of line).

Reply:
Also, `git config --global core.autocrlf false` is helpful for on windows.

Reply:
> Also, `git config --global core.autocrlf false` is helpful for on windows.

this line needs to be included in the file?

Reply:
No it's a local configuration